test/test-util.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * Tools and data for the testing
+ */
+const TestUtil = {
+    getRandomIP: () => {
+        let ip = "";
+        for(let i = 0; i < 4; i++) {
+            ip += Math.floor(Math.random() * 255) + 1 + ".";
+        }
+        return ip.slice(0,-1);
+    },
+
+    getIPInfoTestData: () => {
+        return {
```

Comment:
You could use [faker](https://www.npmjs.com/package/faker) for this one, but it's out of scope of the task.

Reply:
Yes, faker would be good option, and util is not a good name in this case, but let’s leave that for another time :)